packages/stores/src/account/utils.ts

```diff
@@ -90,6 +90,32 @@ export function makeSignDocAmino(
   };
 }
 
+export function getPublicKeyTypeUrl({
+  chainId,
+  chainFeatures,
+}: {
+  chainId: string;
+  chainFeatures: string[];
+}) {
+  const chainIsInjective = chainId.startsWith("injective");
+  const chainIsStratos = chainId.startsWith("stratos");
+  const useEthereumSign = chainFeatures.includes("eth-key-sign");
+
+  if (!useEthereumSign) {
```

Comment:
Is this ! intentional?

Reply:
Yes, this is the default typeUrl. All non `eth-key-sign` transactions require this